### DISCUSSION TITLE
Implement d12frosted/vulpea (enhancements on org roam) in this plugin

### DISCUSSION THREAD

**Discussion by chipsenkbeil** [chipsenkbeil/org-roam.nvim#26] [Ideas]
When skimming https://d12frosted.io/posts/2021-06-11-path-to-org-roam-v2.html, it had a section on [vulpea](https://github.com/d12frosted/vulpea). When org roam migrated to v2, it integrated some core ideas from vuplea into it, but there may still be some features that I'd like that exist in vulpea-only that I'd be comfortable implementing in this plugin.

Saving space here in to write down some of the additional features that vulpea has such as the query engine and capturing that may be nice to have.

**Comment by pejaab** [comment]
I'm currently considering saying goodbye to my emacs setup that I by now only use for org-mode, org-roam and vulpea.
What vulpea adds for me is the "indexing" and resurfacing of org-roam nodes in the agenda that include todos or more specifically the todos from these nodes. This is done via tagging a node as `project`. It has some more tagging functionality to distinguish persons. It's great functionality at least how I make use of org-roam. Would love to see this in the nvim version.
Great work on this btw!

**Reply by chipsenkbeil** [reply]
Hi, can you provide a more detailed breakdown of the ask in terms of missing features or API integration points? Examples visually is a bonus :)

**Reply by pejaab** [reply]
Hey, thanks for the fast reply.
I started out taking screenshots in an attempt to document it thoroughly. Though when looking at my emacs config files I found this [reference](https://www.d12frosted.io/posts/2020-06-23-task-management-with-roam-vol1).
I think I would do injustice to the awesome documentation that d12frosted himself did on what [vulpea](https://github.com/d12frosted/vulpea) enables you to do. You'll have to follow the series of blog entries to see where he's taking this additional functionality. Happy to attempt and clarify further but honestly don't think I can provide anything this detailed and not basically replicate it. But do let me know!
